### PR TITLE
Removed truncation for Operator Installs and added Github Star button

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

### CODE REVIEW

**Comment by gdsoumya** [comment]
Do not update branch, current staging branch is unstable. This PR removes the unstable commits and some other minor changes.

**Comment by chandankumar4** [comment]
> /preview 10
> 
> Link Expires In : 10mins
> 
> * 3.136.2.227:35795

Is the analytics value is not setup for this?

**Comment by gdsoumya** [comment]
> > /preview 10
> > Link Expires In : 10mins
> > 
> > * 3.136.2.227:35795
> 
> Is the analytics value is not setup for this?

No it is missing the aut.json

File: app/client/cypress/integration/CharthubSmallTests.spec.js [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-      .get("a")
-      .each(($div) => {
-        let link = $div.attr("href");
-        let linkName = $div.text();
-        cy.log("Clicking on "+linkName+" link");
-        cy.wrap($div).click();
-      });
-  });
-});
+// describe("Testing the functionality of Footer Links", () => {
```
Comment (chandankumar4) [inline app/client/cypress/integration/CharthubSmallTests.spec.js:9]: Why the testing code itself are commented?

Comment (gdsoumya) [reply app/client/cypress/integration/CharthubSmallTests.spec.js:9]: The test code is wrong, it was the one throwing errors. @Jonsy13 is going to take a look at it after this is merged.

Comment (chandankumar4) [reply app/client/cypress/integration/CharthubSmallTests.spec.js:9]: ok

**Pull request merged**